validphys2/src/validphys/datafiles/new_commondata/CMS_W_13TEV_ETA/metadata.yaml

```
@@ -0,0 +1,98 @@
+# Generalia
+setname: "CMS_W_13TEV_ETA"
```

```suggestion
setname: "CMS_WPWM_13TEV_ETA"
```

And the same for the name of the folder, since it contains both WP and WM